lib/watir/elements/input.rb

```diff
@@ -0,0 +1,15 @@
+module Watir
+  class Input < HTMLElement
+
+    #
+    # Returns applicable label.
+    # Is not lazy loaded.
+    #
+
+    def label
+      el = parent(tag_name: 'form').label(for: id)
+      return el if el.exist?
```

Comment:
@titusfortner, can you explain why we don't want to lazy load the label element?

It seems weird to be returning `nil` when the label element doesn't exist. When there is no associated label, you will get a `NoMethodError`:

~~~~~~~~ruby
browser.radio.label.text
#=> undefined method `text' for nil:NilClass (NoMethodError)
~~~~~~~~

This isn't very descriptive of the problem. I would have expected the usual `UnknownObjectException`.